WIP-CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ Harbor is upgraded to v2.8.1, which drops support for chartmuseum. If you are us
 - Updated harbor chart to `v1.12.1` which upgraded harbor to `v2.8.1`
   - Dropped support for chartmuseum
   - Default login page is now OIDC, for local login refer to path `<url>/account/sign-in`
+- Updated ingress-nginx chart to `v4.7.0` with upgrade ingess-nginx to `v1.8.0`
+  - Added support for kubernetes `v1.24.0`
```

Comment:
Did you mean?
```suggestion
  - Added support for kubernetes `v1.25.0`
```

Reply:
Nope, they only mentioned up to v1.24.0 in there release notes.

Reply:
I looked through the tests now, and version 1.8.0 supports up to 1.27.0. So i maybe just drop that comment. 
Strange that they did not mention it in the release notes.